core/src/main/java/com/linecorp/armeria/common/AbstractRequestContext.java

```diff
@@ -139,10 +156,21 @@ public final ChannelFutureListener makeContextAware(ChannelFutureListener listen
         return RequestContext.super.makeContextAware(future);
     }
 
-    private <T extends Future<?>> void invokeOperationComplete(
-            GenericFutureListener<T> listener, T future) throws Exception {
+    /**
+     * Marks this {@link RequestContext} as having been timed out.
```

Comment:
Could you also add that calling this method will fail-fast the invocation of the callbacks created with `makeContextAware()`?